.github/CHANGELOG.md

```diff
@@ -23,6 +23,29 @@ Ashish Panigrahi
 
 <h3>New features since last release</h3>
 
+* Added a sparse hamiltonian observable and the functionality to support computing its expectation
+  value. [(#1398)](https://github.com/PennyLaneAI/pennylane/pull/1398)
+
+  For example, the expectation value of a sparse hamiltonian, an identity matrix in this example,
+  can be computed as:
+
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev, diff_method="parameter-shift")
+  def circuit(param, H):
+      qml.PauliX(0)
+      qml.SingleExcitation(param, wires = [0, 1])    
+      return qml.expval(qml.SparseHamiltonian(H, wires=[]))
+
+  >>> print(circuit([0.5], scipy.sparse.eye(4).tocoo()))
+  0.9999999999999999
```

Comment:
I recommend never mixing python and pycon syntax -- best to always separate them into separate codeblocks. This is because copy-pasting functionality won't work correctly; the presence of a single `>>>` will make Sphinx think that all other lines are output, and will not be copied.

So something like:

```markdown
  For example, the following QNode returns the expectation value of a sparse Hamiltonian:

  ``python
  dev = qml.device("default.qubit", wires=2)

  @qml.qnode(dev, diff_method="parameter-shift")
  def circuit(param, H):
      qml.PauliX(0)
      qml.SingleExcitation(param, wires = [0, 1])    
      return qml.expval(qml.SparseHamiltonian(H, wires=[]))
  ``
  
  We can execute this QNode, passing in a sparse identity matrix:

  ``pycon
  >>> print(circuit([0.5], scipy.sparse.eye(4).tocoo()))
  0.9999999999999999
  ``
```